sources/packages/web/src/views/Student.vue

```diff
@@ -176,7 +176,7 @@ export default {
     const onSubmit = handleSubmit(async formValues => {
       alert(JSON.stringify(formValues, null, 2));
       // TODO: Remove to enable API Call.
```

Comment:
This TODO is no longer valid i think

Reply:
I've to work on the UI part to finish the story. So, some todos are still pending